src/main/java/org/apache/sling/servlets/post/impl/SlingPostServlet.java

```diff
@@ -234,6 +235,11 @@ protected void doPost(final SlingHttpServletRequest request,
             } catch (ResourceNotFoundException rnfe) {
                 htmlResponse.setStatus(HttpServletResponse.SC_NOT_FOUND,
                     rnfe.getMessage());
+            } catch (final PersistenceException pe) {
+                log.warn("PersistenceException while handling POST "
```

Comment:
at least on DEBUG level the exception should be logged. Otherwise it is close to impossible to debug such response status codes.

Reply:
This WARN statement actually logs the complete stacktrace (see line 241).

Reply:
don't we have redundant information here then? we'll see that SlingPostServlet is logging a PersistenceException, and we'll have a line saying "PersistenceException while handling POST"?